src/modules/event/mutations/CreateEventMutation.ts

```diff
@@ -0,0 +1,48 @@
+import { GraphQLNonNull, GraphQLID } from 'graphql';
+
+import { mutationWithClientMutationId } from 'graphql-relay';
+
+import { errorField, successField, getObjectId } from "@entria/graphql-mongo-helpers";
+
+import EventModel from '../../event/EventModel';
+
+import * as EventLoader from '../../event/EventLoader';
+
+import EventType from '../../event/EventType';
+
+type Args = {
+  event: string;
+};
+const mutation = mutationWithClientMutationId({
+  name: 'Create event',
```

Comment:
wrong name, here is the name of mutation

`CreateEvent`

Reply:
try to keep a pattern like `FeatureAction`

- EventCreate
- EventUpdate
- EventRemove